src/Eval.hs

```diff
@@ -702,9 +707,7 @@ loadInternal ctx xobj path i reloadMode = do
         _ <- liftIO $ setCurrentDirectory cur
         doGitLoad path fpath
       else do
-        _ <- liftIO $ readProcessWithExitCode "git" ["init"] ""
-        _ <- liftIO $ readProcessWithExitCode "git" ["remote", "add", "origin", path] ""
-        (x0, _, stderr0) <- liftIO $ readProcessWithExitCode "git" ["fetch", "--all", "--tags"] ""
+        (x0, _, stderr0) <- liftIO $ readProcessWithExitCode "git" ["clone", "--recurse-submodules", path, "."] ""
```

Comment:
Is this change equivalent?

Reply:
This actually assumes that this is cloned for the first time. It shouldn’t be equivalent?

Reply:
That's what I thought too...

Reply:
That was accidentally included in this PR, I'll send a separate PR to checkout submodules.